Remove repeated clauses beginning with `but`, avoid mentioning JSON (#961)
meilisearch/documentation: learn/getting_started/quick_start.md

# Quick Start

> This quick tour will help you get started with MeiliSearch in only a few steps. In order to follow along, you'll need to be somewhat familiar with the command line. Many of the examples use JavaScript, but you are not required to know JavaScript syntax. An understanding of basic programming concepts will help you make the most of this guide.

Reviewer: ```suggestion
This quick tour will help you get started with MeiliSearch in only a few steps. In order to make the most of it, you'll need to be somewhat familiar with: 

- The command line
- cURL
- JavaScript syntax
```

Reviewer: I suggest that we use a list (easier to skim for experienced developers who just want to get to the MeiliSearch-specific stuff), and that we avoid "basic programming concepts" because it's vague 😄
